torch_glow/src/PyTorchLoaderTest.cpp

```diff
@@ -103,8 +106,9 @@ TEST(PyTorchModelLoader, Model) {
   vec.insert(vec.begin(), module->module_object());
 
   at::ArrayRef<torch::jit::IValue> inputs(vec);
-  PyTorchModelLoader loader(*f, *subgraph, inputs, inputPlaceholders,
-                            outputPlaceholders);
+  errPtr = PyTorchModelLoader::loadJITGraph(
+      *f, *subgraph, inputs, inputPlaceholders, outputPlaceholders);
+  EXPECT_FALSE(!errPtr); // TODO
```

Comment:
What is the TODO here?

Reply:
Currently not all operators are supported by Glow for resnet18, so the failure is expected, but once all operators are implemented, unit test will fail and TODO is:
EXPECT_FALSE(!errPtr); => EXPECT_FALSE(errPtr);